apps/docs/pages/guides/with-nextjs.mdx

```diff
@@ -312,7 +312,14 @@ export default function Account({ session }) {
 </TabPanel>
 <TabPanel id="ts" label="TypeScript">
 
-Let's create a new component for that called `Account.tsx` within a `components` folder.
+We need to create a type definition file for the SQL tables in your project. Create a `utils` folder, and then run the following commands with `PROJECT_ID` replaced with your project's id, which is the same as the subdomain of your `NEXT_PUBLIC_SUPABASE_URL`:
+
+```bash
+npm install --save-dev supabase
+npx supabase gen types typescript --project-id PROJECT_ID > utils/database.types.ts
```

Comment:
This only works if you run two other commands first.
```bash
npx supabase login
npx supabase link
```

Reply:
I agree I missed the login command (and I've addressed that now here), but I didn't use the link command so it doesn't seem strictly necessary, and I'm not clear how to use it: if I run it while following this tutorial, I get an error (`Error: Missing config: open supabase/config.toml: The system cannot find the path specified.`). Following the tutorial never causes that file to be created.

The gen types command seems to work fine without the link command (maybe because I'm using the --project-id flag? It took me a lot of guessing to figure out to do that. All of the docs I found about the gen types command did not use that flag and didn't work for me without it for similar errors as the link command).

If getting that link command working and using it would be useful for something in the tutorial, then 1) could someone else who knows more about it than me help out with that, and 2) assuming there isn't a working solution for that immediately suggested here, could that be treated as an enhancement separate from this PR not blocking this?